nuxeo-document-history-page-provider.html

```diff
@@ -36,14 +36,15 @@
         display: none;
       }
     </style>
-
-    <nuxeo-operation id="auditOp"
-      op="Audit.QueryWithPageProvider"
-      enrichers="{{enrichers}}"
-      schemas="[[schemas]]"
-      headers="{{headers}}">
-    </nuxeo-operation>
-
+    <nuxeo-audit-page-provider id="auditOp" provider-name="DOCUMENT_HISTORY_PROVIDER"
+                               auto="[[auto]]"  auto-delay="[[autoDelay]]"
+                               params="[[params]]" page-size="[[pageSize]]" page="[[page]]"
+                               current-page="{{currentPage}}" sort="{{sort}}"
+                               number-of-pages="{{numberOfPages}}"  results-count="{{resultsCount}}"
+                               is-next-page-available="{{isNextPageAvailable}}"
+                               current-page-size="[[currentPageSize]]"  enrichers="[[enrichers]]"
+                               schemas="[[schemas]]" headers="[[headers]]"  loading="{{loading}}">
```

Comment:
were you referring  to the isNextPageAvaliable? 
That one tells the client if there are more pages to load or not (is not meant to be set by the client, but read by it)
I tried to put the $ on this one and the infinite scroll of the nuxeo-data-table stopped working :(